workers/unity/Packages/com.improbable.gdk.core/Worker/SerializedMessagesToSend.cs

```diff
@@ -112,7 +115,7 @@ public void SendAll(Connection connection, CommandMetaData commandMetaData)
             for (int i = 0; i < updates.Count; ++i)
             {
                 ref readonly var update = ref updates[i];
-                connection.SendComponentUpdate(update.EntityId, update.Update);
+                connection.SendComponentUpdate(update.EntityId, update.Update, updateParams);
```

Comment:
Isn't this a semantic breaking change?

Reply:
Actually that is only half true. We don't reapply the update to the component, but we do add it to the updates received list. So these might need to stay :(. We currently have nothing internal depending on this behaviour but a user might. Basically monobehaviours should have an immediate update or we should keep a list of updates sent as well as received.

Reply:
Hmm will keep the loopback for the moment although we really should get rid of it.